docs/content/latest/architecture/concepts/yb-tserver.md

```diff
@@ -40,17 +40,41 @@ often than others. For example, one table may have a read-heavy usage pattern co
 others. The block cache will automatically favor blocks of this table as the block cache is global
 across all tablet-peers.
 
+## Space amplification
+YugabyteDB's compactions are size tiered. 
+Size tier compactions have the advantage of lower disk write (IO) amplification when compared to level compactions. 
+There's sometimes concern that size-tiered compactions have higher space amplification (that it needs 50% space head room). 
+
+This is not true in YugabyteDB because each table is broken into several tablets and concurrent compactions across 
+tablets are throttled to a certain maximum. Therefore the typical space amplification in YugabyteDB tends to be in the 10-20% range.
+
 ## Throttled compactions
 
 The compactions are throttled across tablets in a given YB-TServer to
 prevent compaction storms. This prevents the often dreaded high foreground latencies during a
 compaction storm.
 
+The default policy makes sure that doing a compaction is worthwhile.
+The algorithm tries to make sure that the files being compacted are somewhat in the similar size ballpark. 
```

Comment:
somewhat in the similar size ballpark
-->
are not too disparate in terms of size.